Resolutions/ChildOfLight_Resolution/rules.txt

```diff
@@ -108,26 +107,28 @@ formats = 0x01a
 overwriteWidth = ($width/$gameWidth) * 512
 overwriteHeight = ($height/$gameHeight) * 512
 
-[TextureRedefine] # Half-res
+[TextureRedefine] # Half-Res
+width = 640
+height = 368
+tileModesExcluded = 0x001
+overwriteWidth = ($width/$gameWidth) * 640
+overwriteHeight = ($height/$gameHeight) * 368
```

Comment:
Not all games use padded textures, is there a reason why you added this? Because if it's not used by the game I'd rather not define it at all.

Reply:
Well, I did notice the game loads 640x368 textures on memory based on my texture dumps, but I can't confirm if it actually renders them, but I thought it would be safer to add them.

Reply:
You can also enable the View Texture Cache Warnings in the Debug->Logging menu to see that there's no problem where a padded layer needs to be added.